src/gui/MainWindow.cpp

```diff
@@ -213,11 +213,9 @@ MainWindow::MainWindow()
 
     setWindowIcon(filePath()->applicationIcon());
     m_ui->globalMessageWidget->setHidden(true);
-    // clang-format off
-	 connect(m_ui->globalMessageWidget, &MessageWidget::linkActivated, &MessageWidget::openHttpUrl);
```

Comment:
I just happened to notice that this was using tabs instead of spaces, for which there doesn't seem to be an obvious reason (rest of the file uses spaces).